### ISSUE TITLE
weForm settings cannot be changed from their defaults

### ISSUE THREAD

**Issue by MDAdler** [weDevsOfficial/weforms#191]
I have a site that has been running weForms with reCaptcha 2 and various custom settings - e.g. Submittion button text and response message, for about three months. Updated to 1.4.3 on Jan 11 and no problems/changes to contact page behaviour. Updated to 1.4.4 on Jan 17, still no problems or changes to the front-end customizations or reCaptcha functionality. Went to add a new contact page with new contact form 4 hours ago and everything fell apart. All of the weForms' setting got reset back to default values as well as reCaptcha returning a false "reCAPTCHA validation failed".

"Submit" button text was reset to "Submit Query". Any attempt to change the Submit button text shows in the preview as being successful but reloading the form shows that the change to the text do not stick.

Any attempt to change the default response text - "Thanks for contacting us! We will get in touch with you shortly." also fails to stick. Previous changes to this text have been reset to the default.

Basically, any changes you attempt to make on the settings tab will not stick, including trying to redirect to a different page etc.

Deactivating the plugin and deleting it should delete all the data associated with the plugin. You even get a warning pop-up to remind you that all the data associated with the plugin will be lost (e.g. form definitions, form submission logs, etc.) but it is not deleted. Reinstalling the plugin revives all the old and corrupted forms.

**Comment by rokibulislam** [comment]
This issue is being raised from wp-user-frontend, I think you're using weforms with wp-user-frontend which is the reason this issue is arising this issue. This issue has been fixed on the wp-user-frontend develop version, so when the next release comes out. you can update the plugin to see the issue has been fixed.
